### PR TITLE
Update freeze_unfreeze callback

### PR DESCRIPTION
**Pull request by Animatory**

### CODE REVIEW

File: torchok/callbacks/freeze_unfreeze.py [hunk 1]
```diff
@@ -1,71 +1,83 @@
...
         """Init FreezeUnfreeze.
 
         Args:
-            epoch2module_names: Incremental unfreeze dictionary. Keys - unfreeze epoch,
-                values - module names to unfreeze. By default, all the modules named in this dictionary
-                will be frozen before training.
-            freeze_bn: If freeze batch norm layers.
+            freeze_modules: List with dictionaries of models to be free-unfreeze.
+            Possible keys of dictionaries are:
+                `module_name` (int): module name relative to the task on which freeze will be applied.
```
Comment (PososikTeam) [inline torchok/callbacks/freeze_unfreeze.py:61]: (str)

File: torchok/callbacks/freeze_unfreeze.py [hunk 2]
```diff
@@ -1,71 +1,83 @@
...
+        if not hasattr(target_module, 'get_stages'):
+            raise ValueError(f"You specified `stages` in `{module_name}` "
+                             f"but this module does not have `get_stages` method")
+        target_module = target_module.get_stages(module_dict['stages'])
+
+    if 'module_class' in module_dict:
+        module_class = module_dict['module_class']
+        filtered_modules = nn.ModuleList()
+        for mod in target_module.modules():
+            parents = [i.__name__ for i in inspect.getmro(type(mod))]
```
Comment (PososikTeam) [inline torchok/callbacks/freeze_unfreeze.py:41]: Cool method)

**Review by PososikTeam — CHANGES_REQUESTED** [review]
Great work! Please change module_name (int) -> module_name (str) in documentation.

File: torchok/callbacks/freeze_unfreeze.py (hunk 1 above)
Comment (Animatory) [reply torchok/callbacks/freeze_unfreeze.py]: fixed

**Review by PososikTeam — APPROVED** [review]

**Pull request merged**